lacommunaute/forum_moderation/tests/test_topic_delete_view.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from django.urls import reverse
+
+from lacommunaute.forum.factories import ForumFactory
+from lacommunaute.forum_conversation.factories import TopicFactory
+
+
+@pytest.fixture
+def topic():
```

Comment:
Petit commentaire en passant ;).
```suggestion
@pytest.fixture(name="topic")
def topic_fixture():
```
Ça permet d'éviter le _shadowing_ de nom (et donc que certain outils soit tout rouge :)) quand tu utilises la fixture ailleurs dans le même fichier.

Reply:
Les tests ne reconnaissent pas `topic` mais bien `topic_fixture` en paramètre. Je n'ai pas retrouvé de doc sur le param `name`. Aurais tu une ref à me suggérer ?
```
@pytest.fixture
def topic_fixture(name="topic"):
    topic = TopicFactory(forum=ForumFactory(with_public_perms=True), with_post=True)
    assign_perm("can_delete_posts", topic.poster, topic.forum)
    return topic
```

Reply:
Le `name` est un paramètre du décorateur pas de la fonction ;).
https://docs.pytest.org/en/7.1.x/reference/reference.html?#pytest.fixture

Reply:
:-O shame on me